python/tests/spark/sql/codegen/test_sklearn_flavor.py

```diff
@@ -88,7 +86,7 @@ def test_sklearn_random_forest(mlflow_tracking_uri: str, spark: SparkSession):
         rikai.mlflow.sklearn.log_model(
             model,
             artifact_path="model",
-            schema="long",
+            model_type="random_forest_classification",
```

Comment:
as a user, how would they know to input this string as the model type?

Reply:
Via document https://github.com/eto-ai/rikai/issues/607